gui/wxpython/location_wizard/wizard.py

```diff
@@ -329,46 +329,43 @@ def __init__(self, wizard, parent):
 
         # toggles
         self.radioEpsg = wx.RadioButton(parent=self, id=wx.ID_ANY, label=_(
-            "Select coordinate reference system by EPSG"), style=wx.RB_GROUP)
+            "Select CRS from a list by EPSG or description"), style=wx.RB_GROUP)
         #self.radioIau = wx.RadioButton(
         #    parent=self, id=wx.ID_ANY,
         #    label=_("Select IAU code of spatial reference system"))
         self.radioFile = wx.RadioButton(
             parent=self, id=wx.ID_ANY, label=_(
-                "Read projection and datum terms from a "
-                "georeferenced data file"))
+                "Read CRS from a georeferenced data file"))
+        self.radioXy = wx.RadioButton(parent=self, id=wx.ID_ANY, label=_(
+                "Create a generic cartesian coordinate system (XY)"))
         self.radioWkt = wx.RadioButton(
             parent=self, id=wx.ID_ANY, label=_(
-                "Read projection and datum terms from a "
-                "Well Known Text (WKT) .prj file"))
-        self.radioSrs = wx.RadioButton(parent=self, id=wx.ID_ANY, label=_(
-            "Select coordinate system parameters from a list"))
+                "Specify CRS using WKT string"))
         self.radioProj = wx.RadioButton(
             parent=self, id=wx.ID_ANY, label=_(
-                "Specify projection and datum terms using custom "
-                "PROJ.4 parameters"))
-        self.radioXy = wx.RadioButton(parent=self, id=wx.ID_ANY, label=_(
-            "Create a generic Cartesian coordinate system (XY)"))
+                "Specify CRS using PROJ.4 string"))
+        self.radioSrs = wx.RadioButton(parent=self, id=wx.ID_ANY, label=_(
+                "Define custom CRS"))
 
         # layout
         self.sizer.SetVGap(10)
-        self.sizer.Add(StaticText(parent=self, label=_("Simple methods:")),
+        self.sizer.Add(StaticText(parent=self, label=_("Select Coordinate Reference System (CRS):")),
```

Comment:
I was suggestion dropping "simple methods" altogether and using "Select Coordinate Reference System (CRS)" as the title of the page.